config/gadi.config

```diff
@@ -29,7 +29,7 @@ process {
   }
 
 withName: 'multiqc' {
-	executor  = 'pbspro'
+	executor  = 'local'
     queue     = 'normal'
```

Comment:
queue, cpus, memory, and time can be deleted here, as local execution will use the resources that main.nf is running on